Makefile

```diff
@@ -9,11 +9,15 @@ format-check:
 	ls src/viam/ur/module/*.*pp | xargs clang-format-19 -i --style=file --dry-run --Werror
 
 configure:
-	cmake -S . -B build -G Ninja
+	cmake -S . -B build -G Ninja $(if $(DISABLE_APPIMAGE),-DVIAM_UR_DISABLE_APPIMAGE=ON)
 
 build: configure
 	cmake --build build --target all -- -j4
-	cmake --install build --prefix build/install/usr
+ifeq ($(DISABLE_APPIMAGE),1)
```

Comment:
The appimage build expects to find things under /usr in the install tree, but we don't need that for an installation that we will package for direct execution. Also, our "prefix" before was a bit of a lie, and it was trying to express what `DESTDIR` actually means. I have no idea why cmake wants DESTDIR as an env var and `--prefix`, but it sure makes me upset.

The prefix for a non appimage install is `/` in the destdir since we want `meta.json` at the root and we don't need layers above `bin` `lib` and friends.